src/utils.ts

```diff
@@ -519,3 +519,9 @@ export async function browserPostEventStream(
     return cancelablePromise as any
   }
 }
+
+export function delay(time) {
```

Comment:
just checking; is there a specific reason you're redefining `delay` here instead of using the well-tested package?

Reply:
Will change it 🙈